src/routes/(admin)/questionbank/single-question/+page.server.ts

```
@@ -0,0 +1,47 @@
+import { get } from 'svelte/store';
```

ESLint

[error] 'get' is defined but never used.

(@typescript-eslint/no-unused-vars)